### DISCUSSION TITLE
无法使用微博登录

### DISCUSSION THREAD

**Discussion by Andy-njupt** [walinejs/waline#1855] [Q&A] [answered]
github，twitter，facebook这三种登录方式都ok的，但是微博登录不起作用；
报错内容：{"errno":400,"message":"400 - {\"error\":\"invalid_grant\",\"error_code\":21325,\"request\":\"/oauth2/access_token\",\"error_uri\":\"/oauth2/access_token\",\"error_description\":\"invalid authorization code:6c1d37a7bfa3613b343e5d7b83d62c06\"}"}
麻烦帮忙看一下

**Comment by lizheming** [comment, marked as answer]
微博的配置暂时有问题

**Reply by Andy-njupt** [reply]
好的，感谢，近期会解决吗

**Reply by lizheming** [reply]
暂时不会，主要是在微博开放平台应用申请上有问题，短时间无法解决

**Reply by Andy-njupt** [reply]
好的 明白

**Reply by lizheming** [reply]
好了，可以再试试

**Reply by Andy-njupt** [reply]
好像还是不能微博登录
